### PR TITLE
Add Dutch translation to the app

### PR DESCRIPTION
**Pull request by SvenDub**
Support Dutch in the app.

---

Before submitting your pull request, consider the following: Does your code...

- [ ] Implement functionality
- [ ] Contain comments/documentation
- [ ] Build without errors
- [ ] Contain and pass unit testing
- [ ] Conform to the Style Guide
- [ ] Have no merge conflicts with ```master```

When you've checked all of the boxes above your code is up for review. If approved it can be merged with the ```master``` branch.

If you want feedback on your work before having finished your code, add ```[WIP]``` to the title of your PR.

### CODE REVIEW

File: android/src/main/res/values-nl/strings.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name">Happy News</string>
+    <string name="title_settings">Instellingen</string>
+    <string name="title_settings_about">Over</string>
+    <string name="title_settings_language">Taal</string>
+    <string name="title_settings_notification">Meldingen</string>
+    <string name="title_settings_sources">Bronnen</string>
+    <string name="title_settings_rate_app">Beoordeel App</string>
+    <string name="about_text">Met Happy News brengen wij jou de positiviteit die jij nodig hebt! Uit onze grote hoeveelheid bronnen halen wij alleen het meest positieve nieuws om jouw dag beter te maken. Je hoeft niet in te logggen, het is compleet gratis en 24/7 beschikbaar! \n\nHappy News heeft:\n - Een constante stroom aan positieve artikelen\n - Vrolijke tweets zorgvuldig door ons gefilterd\n - Inspirerende quotes om jou te motiveren\n\nWij van Happy news hebben als missie om iedereens dag wat positiever te maken. Wij maken een lange werkdag draaglijker door wat vrolijkheid en positiviteit te leveren wanneer jij het nodig hebt!</string>
```
Comment (TvanBronswijk) [inline android/src/main/res/values-nl/strings.xml]: >logggen

loggen

>quotes

citaten

File: android/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
+    <string name="preference_twitter_enabled" translatable="false">twitter_enabled</string>
+    <string name="preference_quotes_enabled" translatable="false">quotes_enabled</string>
+    <string name="preference_articles_enabled" translatable="false">articles_enabled</string>
+    <string name="preference_notifications" translatable="false">notifications</string>
+    <string name="preference_sources" translatable="false">notifications</string>
+    <string name="about_text">With Happy News you get the positivity you need to brighten up the day. We have an extensive amount of sources which we use to bring only the most positive news to your screen. No need to log in, completely free and 24/7 available, We just want to bring positivity to your day. \n\n Happy News features:\n - Constant stream of positive articles\n - Happy tweets based on carefully picked hashtags\n - Inspirational quotes to put some spring in your step\n\n At Happy News we aspire to bring positivity around the world. We want to make a hard days work more enjoyable by bringing some much needed positivity when needed.</string>
```
Comment (TvanBronswijk) [inline android/src/main/res/values/strings.xml]: >, We

Should be ". We" or ", we". Prefer ". We"

Comment (SvenDub) [inline android/src/main/res/values/strings.xml]: Fixed.

@DaanTul We should probably fix this in the Play Store listing as well.

File: android/src/main/res/values-nl/strings.xml (hunk 1 above)
Comment (SvenDub) [inline android/src/main/res/values-nl/strings.xml]: Fixed.

@DaanTul We should probably fix this in the Play Store listing as well.

**Pull request merged**